pkg/azuredisk/nodeserver_test.go

```diff
@@ -327,68 +344,86 @@ func TestNodeStageVolume(t *testing.T) {
 }
 
 func TestNodeUnstageVolume(t *testing.T) {
-	skipIfTestingOnWindows(t)
-	errorTarget := "./error_is_likely_target"
-	targetFile := "./abc.go"
+	errorTarget, err := testutil.GetWorkDirPath("error_is_likely_target")
+	assert.NoError(t, err)
+	targetFile, err := testutil.GetWorkDirPath("abc.go")
+	assert.NoError(t, err)
+
 	tests := []struct {
 		desc        string
 		req         csi.NodeUnstageVolumeRequest
-		expectedErr error
+		expectedErr testutil.TestError
 	}{
 		{
-			desc:        "Volume ID missing",
-			req:         csi.NodeUnstageVolumeRequest{StagingTargetPath: targetTest},
-			expectedErr: status.Error(codes.InvalidArgument, "Volume ID not provided"),
+			desc: "Volume ID missing",
+			req:  csi.NodeUnstageVolumeRequest{StagingTargetPath: targetTest},
+			expectedErr: testutil.TestError{
+				LinuxError:   status.Error(codes.InvalidArgument, "Volume ID not provided"),
+				WindowsError: status.Error(codes.InvalidArgument, "Volume ID not provided"),
+			},
 		},
 		{
-			desc:        "Staging target missing ",
-			req:         csi.NodeUnstageVolumeRequest{VolumeId: "vol_1"},
-			expectedErr: status.Error(codes.InvalidArgument, "Staging target not provided"),
+			desc: "Staging target missing ",
+			req:  csi.NodeUnstageVolumeRequest{VolumeId: "vol_1"},
+			expectedErr: testutil.TestError{
+				LinuxError:   status.Error(codes.InvalidArgument, "Staging target not provided"),
```

Comment:
use a defaultError, similar to https://github.com/kubernetes-csi/csi-driver-smb/blob/36a4dbce197fa7dcb82b27d5548479c21b67c9e4/pkg/smb/nodeserver_test.go#L374, otherwise need to write twice for every test error

Reply:
@andyzhangx The issue with that is that in some cases, Windows has a `nil` error. In that case, we need an explicit mention. For example - https://github.com/kubernetes-sigs/azuredisk-csi-driver/blob/430a46a685be481a46b546843d697f8378b5fcbc/pkg/azuredisk/nodeserver_test.go#L167

Reply:
if it's not an error case, we could skip that case, e.g. https://github.com/kubernetes-csi/csi-driver-smb/blob/36a4dbce197fa7dcb82b27d5548479c21b67c9e4/pkg/smb/nodeserver_test.go#L474

otherwise need to copy the error message for every case, that's a burden to write a test.

Reply:
Makes sense. I have added the relevant change. Thanks